src/components/utils/src/threads/thread_qt.cc

```diff
@@ -36,6 +36,18 @@
 
 #include <QtConcurrent/QtConcurrent>
 
+namespace {
+
+// AN: Temporary workaround for Windows platform.
+// Thread ID getting should be reworked to be crossplatform
+uint64_t ConvertQtThreadHandleToUInt64(Qt::HANDLE handle) {
+  const DWORD* win_thread_id =
+      reinterpret_cast<DWORD*>(QThread::currentThreadId());
```

Comment:
@pestOO I don't understand you. `QThread::currentThreadId()` returns pointer to `Qt::HANDLE`